troubleshooting/Hardware_Discovery.md

```diff
@@ -0,0 +1,79 @@
+### Hardware Discovery
```

Comment:
- This file should be moved to the "troubleshooting/known_issues" subdirectory
- Change this to a top-level header `# Hardware Discovery`

Reply:
Completed